plugins/autolink/plugin.js

```diff
@@ -41,7 +41,7 @@
 			editor.on( 'contentDom', function() {
```

Comment:
Technically this surrounding code is wrong, as it creates more than needed listeners. Every `contentDom` event (each time the editable is created) a new `editor#key` listener is added (on top of existing one).

`editor#key` don't need to wait for `contentDom` evt, so it's safe to remove it. Instead we redundant events being fired.

It's not the subject of this issue, but a minor thing that can be fixed along.